lib/media/presentation_timeline.js

```diff
@@ -361,6 +361,13 @@ shaka.media.PresentationTimeline = class {
   getSegmentAvailabilityEnd() {
     if (!this.isLive() && !this.isInProgress()) {
       // It's a static manifest (can also be a dynamic->static conversion)
+      if (this.maxSegmentEndTime_ && this.duration_) {
```

Comment:
Do we care about the possibility of `duration_` or `maxSegmentEndTime_` being 0 here?

If not, you don't need to check if `duration_` is truthy, because it's a non-nullable number and it initializes to Infinity.

Reply:
Ah, true.  Fixed.